e2e/test-app-v9/.gitignore

```diff
@@ -41,3 +41,10 @@ src/__tests__
 # System Files
 .DS_Store
 Thumbs.db
+.yarn/*
+!.yarn/cache
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
why are these excluded?

Reply:
I followed the instruction from Yarn 2 documentation so I wasn't sure ><

Reply:
Btw, i see Jest repo excludes some of the ones above too

Reply:
Can you point me to the docs stating that?

Reply:
Ok, found https://yarnpkg.com/getting-started/qa/#which-files-should-be-gitignored. I'm good then